sdgym/synthesizers/generate.py

```diff
@@ -219,69 +167,4 @@ def sample_from_synthesizer(self, synthesizer):
             return sample_from_synthesizer_fn(synthesizer)
 
     NewSynthesizer.__name__ = f'Custom:{display_name}'
-
-    return NewSynthesizer
-
-
-def create_sequential_synthesizer(
-    display_name, get_trained_synthesizer_fn, sample_from_synthesizer_fn
-):
-    """Create a new sequential synthesizer.
-
-    Args:
-        display_name(string):
-            A string with the name of this synthesizer, used for display purposes only when
-            the results are generated
-        get_trained_synthesizer_fn (callable):
-            A function to generate and train a synthesizer, given the real data and metadata.
-        sample_from_synthesizer (callable):
-            A function to sample from the given synthesizer.
-
-    Returns:
-        class:
-            The synthesizer class.
-    """
```

Comment:
Why are we removing all this ?

Reply:
The last point of the issue mentions removing the sequential code for now